setup.json

```diff
@@ -27,7 +27,7 @@
     ],
     "reentry_register": true,
     "install_requires": [
-        "aiida-core>=1.0.1,<2.0.0",
+        "aiida-core>=1.4.0,<2.0.0",
```

Comment:
Do we have to change this? Or does the current release still work with aiida-core 1.0.1?

Reply:
If I remember correctly, there was a change in FolderData.open() method: 'key' input was renamed to 'path'. Will it work for older versions then? I have to tested it

Reply:
You are right. by adapting to path I prob broke the aiida-core back compatibility. Otherwise we would get tons of warnings. So we just leave it to the 1.4.0 requirement

Reply:
Have you tried a fresh installation ones to make sure that we do not clash with any other aiida-core dependencies, or is the CI env fresh and we do not need to test it?
Anyway we should test the the 'pip install aiida-fleur' after the pypi upload. From my side we are good to go.

Reply:
Looks like a fresh installation did not lead to conflicts.